src/test/java/seedu/address/model/attendance/AttendancePresenceTest.java

```diff
@@ -0,0 +1,103 @@
+package seedu.address.model.attendance;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+public class AttendancePresenceTest {
```

Comment:
might want to test for white spaces after present to cover the trim() part in your method
" String validPresence= "present     ";

Reply:
Sounds good! Will add that, thanks!